src/Microsoft.AspNet.SignalR.Core/TaskAsyncHelper.cs

```diff
@@ -1354,5 +1354,24 @@ private static class TaskCache<T>
         {
             public static Task<T> Empty = MakeTask<T>(default(T));
         }
+
+        [SuppressMessage("Microsoft.Performance", "CA1811:AvoidUncalledPrivateCode", Justification = "This is a shared file")]
+        public static void Dispatch(Action action)
```

Comment:
OK. 
On a different note - the suppression should be removed since it is no longer correct/needed.